crates/api_models/src/payments.rs

```diff
@@ -1636,12 +1644,7 @@ pub struct OrderDetails {
 pub struct Metadata {
     /// Information about the product and quantity for specific connectors. (e.g. Klarna)
     pub order_details: Option<OrderDetails>,
-    /// Information used for routing
-    pub routing_parameters: Option<HashMap<String, String>>,
```

Comment:
Why was this field removed, is it no longer required? If it is required, then how are we accepting it?

Reply:
Yeah I have confirmed with @bernard-eugine and @vspecky . It can be removed.